docs/misc/changelog.rst

```diff
@@ -35,6 +35,7 @@ Others:
 
 Documentation:
 ^^^^^^^^^^^^^^
+- Add plotting to the Monitor example
```

Comment:
Feel free to add the change from previous PR as well, along with your nick :)

Reply:
> add the change from previous PR 

I did that in #508 ;)